guide/src/class.md

```diff
@@ -53,7 +53,7 @@ impl MyClass {
 
      #[new]
      fn __new__(obj: &PyRawObject, num: i32) -> PyResult<()> {
-         obj.init(||   {
+         obj.init(   {
```

Comment:
Just a nit, but it looks redundant spaces

Reply:
Thanks, fixed